.test-infra/jenkins/job_PostCommit_Website_Merge.groovy

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import PostcommitJobBuilder
+
+
+// This job builds and merges the website.
+PostcommitJobBuilder.postCommitJob('beam_PostCommit_Website_Merge', 'Run Website Merge',
+  'Website Merge', this) {
+
+  description('Builds and merges the web site.')
```

Comment:
A user without prior knowledge might not understand what a 'Website Merge' means. How about elaborating a bit in the description? i.e.:

    'Merge generated website content into asf-site branch for hosting'